public/distinctions.json

```
@@ -0,0 +1,23 @@
+{
+  "Paris Peace Forum": {
+    "description": {
+      "fr": "Sélectionné comme solution par le Forum de Paris sur la paix 2020",
+      "en": "Selected as Solution by the Paris Peace Forum 2020"
+    },
+    "link": "https://parispeaceforum.org/projects/community-against-disinfo/"
+  },
+  "Nobel Prize Summit": {
+    "description": {
+      "fr": "Présenté comme une «innovation de source ouverte pour l'intégrité de l'information» au sommet du prix Nobel 2023",
+      "en": "Featured as “Open-Source Innovation for Information Integrity” at the Nobel Prize Summit 2023"
+    },
+    "link": "https://digitalpublicgoods.net/information-pollution/"
```

For those browsers that support it, let's use [Fragment Text Directives](https://wicg.github.io/scroll-to-text-fragment/) to scroll the page to our description 🙂 
```suggestion
    "link": "https://digitalpublicgoods.net/information-pollution/#:~:text=Open%20Terms%20Archive,digital%20experience."
```

---

woooo, i didn't know the URL Fragment Text Directives!